.github/workflows/merge_stage_test.yml

```diff
@@ -0,0 +1,201 @@
+        run: pip install Pillow==6.2.2
+        if: ${{matrix.torchvision == '0.4.2'}}
+      - name: Install PyTorch
+        run: pip install torch==${{matrix.torch}}+cpu torchvision==${{matrix.torchvision}}+cpu -f https://download.pytorch.org/whl/torch_stable.html
+      - name: Install mmseg dependencies
+        run: |
+          python -V
+          pip install -U openmim
+          pip install git+https://github.com/open-mmlab/mmengine.git
+          mim install 'mmcv>=2.0.0rc1'
+          pip install git+https://github.com/open-mmlab/mmclassification.git@dev-1.x
+      - name: Install unittest dependencies
+        run: pip install -r requirements/tests.txt -r requirements/optional.txt
+      - name: Build and install
+        run: rm -rf .eggs && pip install -e .
+      - name: Run unittests and generate coverage report
+        run: |
+          pip install timm
+          coverage run --branch --source mmseg -m pytest tests/
+          coverage xml
+          coverage report -m
+        if: ${{matrix.torch >= '1.6.0'}}
+      - name: Skip timm unittests and generate coverage report
+        run: |
+          coverage run --branch --source mmseg -m pytest tests/ --ignore tests/test_models/test_backbones/test_timm_backbone.py
+          coverage xml
+          coverage report -m
+        if: ${{matrix.torch < '1.6.0'}}
+  build_cpu_pt:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: [3.7]
+        torch: [1.6.0, 1.7.1, 1.8.1, 1.9.1, 1.10.1, 1.11.0]
+        include:
+          - torch: 1.6.0
+            torchvision: 0.7.0
+          - torch: 1.7.1
+            torchvision: 0.8.2
+          - torch: 1.8.1
+            torchvision: 0.9.1
+          - torch: 1.9.1
+            torchvision: 0.10.1
+          - torch: 1.10.1
+            torchvision: 0.11.2
+          - torch: 1.11.0
+            torchvision: 0.12.0
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Upgrade pip
+        run: pip install pip --upgrade
+      - name: Install Pillow
+        run: pip install Pillow==6.2.2
+      - name: Install PyTorch
+        run: pip install torch==${{matrix.torch}}+cpu torchvision==${{matrix.torchvision}}+cpu -f https://download.pytorch.org/whl/torch_stable.html
+      - name: Install mmseg dependencies
+        run: |
+          python -V
+          pip install -U openmim
+          pip install git+https://github.com/open-mmlab/mmengine.git
+          mim install 'mmcv>=2.0.0rc1'
+          pip install git+https://github.com/open-mmlab/mmclassification.git@dev-1.x
+      - name: Install unittest dependencies
+        run: pip install -r requirements/tests.txt -r requirements/optional.txt
+      - name: Build and install
+        run: rm -rf .eggs && pip install -e .
+      - name: Run unittests and generate coverage report
```

Comment:
I think it should follow ocr setting that only upload ut report on  python3.7 && pytorch1.8.1

https://github.com/open-mmlab/mmocr/blob/7f3d832074168c0d2665e728f0fe95da99b72681/.github/workflows/merge_stage_test.yml#L105-L107

Reply:
OK.